SmartDeviceLink/SDLChoiceCell.h

```diff
@@ -0,0 +1,82 @@
+//
+//  SDLChoiceCell.h
+//  SmartDeviceLink
+//
+//  Created by Joel Fischer on 5/21/18.
+//  Copyright © 2018 Livio. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class SDLArtwork;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLChoiceCell: NSObject
```

Comment:
It would be handy to have a `cellSelectedHandler` on the `SDLChoiceCell`. That way, when the `SDLChoiceSetDelegate`'s `choiceSet:didSelectChoice:withSource` is called, the handler can be triggered. Otherwise, I have to check the cell's `text` and figure out what I'm going to do based on that... which can get messy.

Reply:
The problem with blocks is that we'd have to store all of them...and there could be a lot. For now, I've added a row index to the delegate callback, which should be sufficient. If we want to go to blocks, we can do so in a later release.